### PR TITLE
chore(release): 8.2.0

### PR DESCRIPTION
**Pull request by xDivisionByZerox**

### CODE REVIEW

**Comment by ST-DDT** [comment]
Are there no changes to the pnpm-lock file?

**Comment by xDivisionByZerox** [comment]
I repeated the release steps and got infact changes in the lock file this time. They must have been disappeared due to an error in the `clear` script I got during the first run.

Problem is that these changes introduce the same problems as in #2184.

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
+
```
Comment (ST-DDT) [inline pnpm-lock.yaml]: You can fix that by manually reverting these.

**Review by ST-DDT — APPROVED** [review]

File: pnpm-lock.yaml (hunk 1 above)
Comment (xDivisionByZerox) [reply pnpm-lock.yaml]: I see. I thought they would be put  in again when the CI runs.

**Review by prisis — APPROVED** [review]

**Pull request merged**